compilers.md

```diff
@@ -164,6 +164,8 @@ Microsoft .NET. FTN95 ships with the world's best runtime checking and a great
 range of supporting software. All standard and many vendor-specific legacy
 language features are supported, so that Fortran projects may be any combination
 of Fortran 77, Fortran 90 and Fortran 95.
+Some features of Fortran 2003 and 2008 have been [added](https://www.silverfrost.com/19/ftn95/support/ftn95_revision_history.aspx).
+There is a free personal edition.
```

Comment:
Could you add the information about the supported platforms and architectures if those are available?

Reply:
According to this page https://www.silverfrost.com/33/ftn95/faq.aspx

> Silverfrost Fortran runs on both 32 and 64-bit Windows. 
> …
> There is only one version of Silverfrost Fortran and it runs on 64-bit Windows just as happily as 32-bit.
> …
> If your programs need huge amounts of memory then turning on 64-bit mode will help you.

I will include the first sentence to this PR.